flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkInputFormatReaderDeletes.java

```diff
@@ -107,7 +107,7 @@ protected StructLikeSet rowSet(String name, Table testTable, String... columns)
     RowType rowType = FlinkSchemaUtil.convert(projected);
     CatalogLoader hiveCatalogLoader = CatalogLoader.hive(catalog.name(),
         hiveConf,
-        hiveConf.get(HiveConf.ConfVars.METASTOREURIS.varname),
+        hiveConf.get(HiveConf.ConfVars.METASTOREURIS.varname), null,
```

Comment:
NIT: use `hiveConf.get(HiveConf.ConfVars.METASTOREWAREHOUSE.varname)` instead of null.